Functions.Templates/Templates/SqlInputBinding-CSharp/.template.config/template.json

```diff
@@ -0,0 +1,66 @@
+{
+    "author": "Microsoft",
+    "classifications": [
+        "Azure Function",
+        "Input",
+        "SQL"
+    ],
+    "name": "SqlInputBinding",
+    "identity": "Azure.Function.CSharp.SqlInputBinding",
+    "groupIdentity": "Azure.Function.SqlInputBinding",
+    "shortName": "sqlinput",
+    "tags": {
+        "language": "C#",
+        "type": "item"
+    },
+    "sourceName": "SqlInputBindingCSharp",
+    "symbols": {
+        "connectionStringSetting": {
+            "description": "The name of the app setting containing your SQL connection.",
+            "type": "parameter",
+            "defaultValue": "",
+            "replaces": "SqlConnectionString"
+        },
+        "object": {
+            "description": "This is the object name for the SQL input binding.",
+            "type": "parameter",
+            "defaultValue": "[dbo].[table1]",
+            "replaces": "object"
+        }
+    },
+    "primaryOutputs": [
+        {
+            "path": "SqlInputBindingCSharp.cs"
+        }
+    ],
+    "defaultName": "SqlInputBindingCSharp",
+    "postActions": [
+        {
+          "Description": "Adding Reference to Microsoft.Azure.WebJobs.Extensions.Sql Nuget package",
+          "ActionId": "B17581D1-C5C9-4489-8F0A-004BE667B814",
+          "ContinueOnError": "true",
+          "ManualInstructions": [],
+          "args": {
+            "referenceType": "package",
+            "reference": "Microsoft.Azure.WebJobs.Extensions.Sql", "version": "1.0.0-preview3",
```

Comment:
TODO: update package version when public preview version is ready

Reply:
@lucyzhang929 Make sure you're updating these package versions. Can you see if there's support for wildcard/semver placeholders? Would be nice not to have to keep updating this every time we released a new version.